File: Group 47/level 20/Homework/Task 02/hw.py
# დავალება 2 :
# შექმნი სია სადაც იქნება სადაც იანქბა ციფრები რომელიც გამოიტანს 5 სა და 3ის ჯერადევბს

num_arr = []

number = 1

while number <= 100:
    if number % 3 == 0 or number % 5 == 0:
       num_arr.append(number)  
    number += 1
print("five and three multiply",num_arr)

File: Group 57/level 120/homework/hw.js
const word = document.getElementById('textInput');
const yellowBtn = document.getElementById("one");
const greenBtn = document.getElementById("four");
const redBtn = document.getElementById("three");
const blueBtn = document.getElementById("two");
const orangeBtn = document.getElementById("five");
const purpleBtn = document.getElementById("six");
const cyanBtn = document.getElementById("seven");
const brownBtn = document.getElementById("eight");



yellowBtn.addEventListener('click', () => {
    word.style.color = "yellow"
})
blueBtn.addEventListener('click', () => {
    word.style.color = "blue"
})
redBtn.addEventListener('click', () => {
    word.style.color = "red"
})
greenBtn.addEventListener('click', () => {
    word.style.color = "green"
})
orangeBtn.addEventListener('click', () => {
    word.style.color = "orange"
})
purpleBtn.addEventListener('click', () => {
    word.style.color = "purple"
})
cyanBtn.addEventListener('click', () => {
    word.style.color = "cyan"
})
brownBtn.addEventListener('click', () => {
    word.style.color = "brown"
})